src/clj/hypertext_db/graph.clj

```diff
@@ -111,7 +112,14 @@
   (-> vault
       (assoc ::nodes {})
       (assoc ::backlinks {})
-      (assoc ::parsers [])))
+      (assoc ::parser-chain [])))
+
+(s/fdef set-parsers
+  :args (s/cat :graph ::t :parsers ::parser-chain)
+  :ret  ::t)
+(defn set-parsers
+  [graph parsers]
+  (assoc graph ::parser-chain parsers))
```

Comment:
- [ ] Unify the constructors. I postponed this because I didn't want to refactor the `vault->` constructor right away.

Reply:
Added ticket #15 to keep track of this